jwql/website/apps/jwql/data_containers.py

```diff
@@ -907,11 +921,12 @@ def get_thumbnails_all_instruments(parameters):
         # Query MAST for all rootnames for the instrument
         service = "Mast.Jwst.Filtered.{}".format(instrument)
 
-        if (parameters['apertures'][inst.lower()] == []) and (parameters['detectors'][inst.lower()] == []) \
-            and (parameters['filters'][inst.lower()] == []) and (parameters['exposure_types'][inst.lower()] == []) \
-            and (parameters['read_patterns'][inst.lower()] == []):
-                params = {"columns": "*",
-                          "filters": []}
+        if ((parameters['apertures'][inst.lower()] == [])
```

Comment:
This is much more readable, thank you for doing this.

Reply:
PEP8 still doesn't like it. But it does match an example given in the PEP8 documentation...